components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -736,4 +737,28 @@ private void loadCodeForRequire() {
                     "adaptive authentication scripts.", e);
         }
     }
+    @Reference(
+            name = "identity.user.profile.mgt.component",
```

Comment:
Shall we change the name indicating the federated association manager?

Reply:
This is the name that was given to the Federated association manager when registering the service. We need to refer the service from the same name